Electronic Trash Onboard Grant SahilD 2.0

sahilchess opened the pull request:
## Submission Checklist:

<!--To check the checkboxes, simply replace the empty space in the brackets with a captial X, like this: [ ] becomes [X]-->

- [X] I am a current high school, middle school, or home schooled student.
  - [X] I am 18 or under
  - [X] I have filled out the [verification form](https://verify.onboard.hackclub.com) **(do this for every pull request)**
- [X] I have joined the [`#electronics`](https://hackclub.slack.com/archives/C056AMWSFKJ) channel on [Slack](https://hackclub.com/slack/?event=onboard)
- [X] I made this board from scratch, even if I followed a tutorial
- [X] I have followed the directions in [README.md](https://github.com/hackclub/OnBoard/blob/main/README.md#getting-started)
  - [X] Created a folder under `onboard/projects`
  - [X] Filled out `TEMPLATE.md` as `README.md` inside your project folder
  - [X] Uploaded `gerber.zip` and `schematic.pdf` inside your project folder
  - [X] Created a folder called `src` and uploaded design files
  - [X] Uploaded screenshot of PCB order called `cart.png` with all costs included
  - [X] I am ordering the smallest batch size (ie. JLCPCB's smallest order is for 5 boards)
  - [X] If outside the US I've checked that I can afford the customs charges in my country (which isn't covered by the grant)
___
- [X] *(Optional)* This project is from a tutorial[^1].
- [ ] *(Optional)* This project is for OnBoard Live[^2].
- [ ] *(Optional)* I'm in a FIRST (FRC, FTC, FLL, etc.) team. The number is: ____

[^1]: Projects from a tutorial are 100% fine! We just want to ask so we can count how people are using tutorials.
[^2]: OnBoard Live is a special version of OnBoard where you can earn more money for designing advanced boards. Check out the #onboard-live channel on [our Slack](https://hackclub.com/slack/?event=onboard)!
<!-- -Submission- -->

prophetorpheus commented:
<!-- MY-ONBOARD-BOT --> Hi, I'm Orpheus Leap! Here to help you review your PR.
  
**projects/trash_sahil:**

Required files
||||
|---|---|---|
|✅| README.md | A description of your project |
|❌ | cart.png | You need to include a screenshot of your JLCPCB. Check out [these instructions](https://github.com/hackclub/OnBoard/blob/main/docs/ordering_from_JLCPCB.md#pcb-review). If you already have one, make sure it's a PNG file named exactly "cart.png". |
|✅| gerber.zip | |
|✅ | [schematic.pdf](<https://raw.githubusercontent.com/hackclub/OnBoard/ab6a513d9091c0e2a156df5b1eca47c91cd15722/projects/trash_sahil/schematic.pdf>) | Manually check [schematic.pdf](<https://raw.githubusercontent.com/hackclub/OnBoard/ab6a513d9091c0e2a156df5b1eca47c91cd15722/projects/trash_sahil/schematic.pdf>) |
|✅| Source files - EasyEDA |  Found both PCB and SCH files |

You can view a render of your board over on [gerber.zip/2d](<https://gerber.zip/2d/?mode=layers&boardUrl=https://raw.githubusercontent.com/hackclub/OnBoard/ab6a513d9091c0e2a156df5b1eca47c91cd15722/projects/trash_sahil/gerber.zip>)!
		
  Happy OnBoarding!
 <sub>This bot is a simpler helper for common submission types. If there's an error with this, please point it out and someone from the OnBoard team will look at it manually.</sub>

grimsteel commented:
Could you delete these three files:

![image](https://github.com/user-attachments/assets/01412cf5-a967-43c3-92d3-3ee8f4111b9e)

sahilchess commented:
okay

grimsteel commented:
It looks like you deleted the entire folder:
![image](https://github.com/user-attachments/assets/447ecd9e-083b-4fe5-b094-e497b0673d44)

You only needed to delete those three files specifically

sahilchess commented:
I don't know how to add it back...

grimsteel commented:
I went ahead and deleted the files for you. Can you update your cart.png to either show the JLCPCB checkout page **with shipping costs**, or the post-checkout order status page?

sahilchess commented:
Done.

sahilchess commented:
Also, I see that you added a no HCB needed tag. Does that mean I don't get a grant?

grimsteel commented:
> Also, I see that you added a no HCB needed tag. Does that mean I don't get a grant?

That tag is just to make it clear that this is not your first submission. I'm still going to reissue your grant.

grimsteel commented:
You still need to add shipping costs to `cart.png`

sahilchess commented:
I did. This is what I see.

![image](https://github.com/user-attachments/assets/078b51f7-8537-4cd4-a9b0-e27616b2c09d)

grimsteel commented:
https://github.com/hackclub/OnBoard/pull/1592#issuecomment-2764738109

Can you take a screenshot of the cart page? It'll be more accurate than that page. Also, I would recommend using a cheaper shipping option

sahilchess commented:
I uploaded a screen shot of the cart page.

grimsteel commented:
Since you're only assembling the top side, can you switch to Economic PCBA? Also, I would recommend choosing a cheaper shipping option

sahilchess commented:
One of the parts is only available in Standard PCBA. My bad, I don't see that you could change the shipping options.

sahilchess commented:
I added the updated cart.

grimsteel commented:
Great! Could you fill out the verification form again for this PR?

sahilchess commented:
Yep! Done. Thanks for reviewing my pr and helping me out!

grimsteel commented:
Hey, I just noticed that this board is from a tutorial, so you get only $50 in your grant (if you make another board you can get the remaining $50).

It looks like the standard only component is your 555 timer - I would recommend switching it out with `C7593` - you'll need to change the footprint but the pinout is the same so you won't need to do any rerouting.

Also, just in general, I would recommend looking for parts (especially resistors and capacitors) in the [basic parts library](https://jlcpcb.com/parts/basic_parts). You'll save a ton of money in extended component fees.

sahilchess commented:
Yep, I change most parts to basic, and nothing comes as standard PCBA. It is significantly cheaper and I also changed the 555 timer. I also changed the USB C to USB A because the pins are farther apart and it makes it easier to wire. I updated all the files.

grimsteel commented:
Great! I've just sent you your grant: https://hcb.hackclub.com/grants/lYIoyp

Good luck getting this manufactured!

grimsteel commented:
>  Thanks! Also, could I get my other card reissued?

The grant I just sent you is basically a reissue of your old card - you didn't use any of it, so you get the full $50 for this board. If you want to manufacture your old submission (the NFC card) let me know and I can top up the current active grant to the full $100.

sahilchess commented:
Yep! That would be great!
